index.d.ts

```diff
@@ -45,5 +45,7 @@ interface Options {
 }
 
 export declare function createEpicMiddleware<T, S>(rootEpic: Epic<T, S>, options?: Options): EpicMiddleware<T, S>;
+export declare function createEpicMiddleware<E, T, S>(rootEpic: E, options?: Options): EpicMiddleware<T, S>;
```

Comment:
Is this overload necessary? Shouldn't the rootEpic always be `(action$, store)` only?